docs/prowjobs.md

```diff
@@ -4,12 +4,13 @@ The Prowjobs in this repo are located under the [jobs folder](../jobs) and are s
 
 ## Job parameters
 
-Each Prowjob is represented as a well-defined set of parameters which are fed as input to a YAML template to generate the final Prowjob YAML. The parameter set is unmarshaled into the following Go struct that corresponds to the standard Prowjob definition fields :
+Each Prowjob is represented as a well-defined set of parameters, which are fed as input to a YAML template to generate the final Prowjob YAML. The parameter set is unmarshalled into the following Go struct that corresponds to the standard Prowjob definition fields :
 ```go
 type JobConfig struct {
 	JobName            string         `json:"jobName,omitempty"`
 	RunIfChanged       string         `json:"runIfChanged,omitempty"`
-	Branches           []string       `json:"branches,omitempty"`
+    SkipIfOnlyChanged  string         `json:"skipIfOnlyChanged,omitempty"`
```

Comment:
Indentation needs to use tags

Reply:
Can we also document this in the table below? I don't think anyone adding a new Prowjob will need this but I feel it should still be in the documentation.

Reply:
yeah for sure

Reply:
You have no idea the struggle I just went through. But the tabs are fixed